### PR TITLE
Add in-hands for the energy stave

### PR DESCRIPTION
**Pull request by bloxgate**
Adds in-hand sprites for the Arbiter's energy stave

Todo:
- [x] West direction states

### CODE REVIEW

File: code/modules/halo/misc/SyetenArbiter.dm [hunk 1]
```diff
@@ -36,6 +36,10 @@
 	icon = 'icons/syetenarbiter.dmi'
 	icon_state = "dogler-staff_obj"
 	item_state = "dogler-staff_obj"
+	item_icons = list(
+		slot_l_hand_str = 'code/modules/halo/misc/SyetenArbiter/inhands_left.dmi',
+		slot_r_hand_str = 'code/modules/halo/misc/SyetenArbiter/inhands_right.dmi',
```
Comment (caelaislinn) [inline code/modules/halo/misc/SyetenArbiter.dm]: This comma at the end of line 41 will create a null entry in the list; it's practically no problem but stylistically bad form

File: code/modules/halo/misc/SyetenArbiter/SyetenArbiter.dm [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+/obj/item/weapon/melee/energy/energystave
+	name = "Energy Stave"
+	desc = "An energy stave with a shimmering blade."
+	icon = 'icons/syetenarbiter.dmi'
+	icon_state = "dogler-staff_obj"
+	item_state = "dogler-staff_obj"
+	force = 65
+	throwforce = 12
+	edge = 0
+	sharp = 0
```
Comment (caelaislinn) [inline code/modules/halo/misc/SyetenArbiter/SyetenArbiter.dm]: This dm file already exists at code/modules/halo/misc/SyetenArbiter.dm, why did you duplicate it here?

**Review by caelaislinn — CHANGES_REQUESTED** [review]
Why did you [ci skip] this?

**Comment by bloxgate** [comment]
The second commit was ci skipped since it didn't touch any code and only added a state to an icon.

File: baystation12.dme [hunk 3]
```diff
@@ -1554,6 +1554,7 @@
 #include "code\modules\halo\misc\armourspecials\shields.dm"
 #include "code\modules\halo\misc\armourspecials\speed.dm"
 #include "code\modules\halo\misc\armourspecials\strength.dm"
+#include "code\modules\halo\misc\SyetenArbiter\SyetenArbiter.dm"
```
Comment (caelaislinn) [inline baystation12.dme:1556]: This is giving a compile error now due to nonexistent file

**Review by caelaislinn — CHANGES_REQUESTED** [review]

**Comment by caelaislinn** [comment]
As it was one of your initial commits in the PR, it meant Travis didn't run at all. Travis does run tests on icons and icon states as well

**Comment by bloxgate** [comment]
That wasn't one of the initial commits. Travis had already ran on the previous commit, as can be seen by the passing marker [here](https://gyazo.com/701e59266a28ac4f204a0ed96e54c194)

**Review by caelaislinn — APPROVED** [review]

**Pull request merged**